src/index.tsx

```diff
@@ -1,13 +1,28 @@
 import { createRoot } from 'react-dom/client';
-import { HashRouter as Router } from 'react-router-dom';
 
 import 'bulma/css/bulma.css';
 import '@fortawesome/fontawesome-free/css/all.css';
 
 import { App } from './App';
+import { HashRouter, Navigate, Route, Routes } from 'react-router-dom';
+import { HomePage } from './components/HomePage';
+import { PeoplePage } from './components/PeoplePage';
+import { NotFoundPage } from './components/NotFound';
 
 createRoot(document.getElementById('root') as HTMLDivElement).render(
-  <Router>
-    <App />
-  </Router>,
+  <html className="has-navbar-fixed-top">
```

Comment:
Using `<html>` as a React component is not standard practice. Typically, React components are rendered within a `<div>` or other container element inside the body of the HTML document. Consider replacing `<html>` with a `<div>` or another appropriate container element.